app/services/twitter_service.rb

```diff
@@ -50,12 +51,17 @@ def self.check_response(response)
   end
 
   def self.valid_recording(code)
-    unless code == ''
+    if not Settings.features.twitter.notifications.enabled == true
```

Comment:
change to unless?  and why should recording validity be influenced by whether twitter notifications are enabled?